uiowa/uiowa
# Events block: Include last_day if end_day not present

Resolves #5916 

Essentially, if an event is marked as All Day and spanning across more than one date, it currently only displays the start date when pulled into a Sitenow Events block.





# How to test


Set up an Events block which features an All Day event over multiple days.
https://content.uiowa.edu/uicb-echizen can be used as an example. (for instance, set up an Events block filtered on Department: Graduate College and Type: Exhibit)

See that rather than only displaying one day,
<img width="1333" alt="image" src="https://user-images.githubusercontent.com/49074566/199784581-2f37fdba-0aa4-4161-91f5-61f3f64c29d5.png">

all days in the range are displayed.
<img width="1367" alt="image" src="https://user-images.githubusercontent.com/49074566/200915619-a09e10ae-02e7-4039-b944-c48866dd4120.png">

Note: 
In looking into this, the above Event is not set up in the recommended practice:

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/49074566/200915776-aff3368f-e9ab-4c3c-9ee5-ac471839b7a7.png">
"If the event spans multiple days, it is recommended to use a date repeat rule instead of different start/end dates. For example, if the event runs all week, select the start date, omit the end date and then add a daily repeat rule that stops on the last day of that week."
But as it's a recommendation and not a requirement, I think this change works as a small fallback.
